src/styles/shared/_buttons.scss

```diff
@@ -180,6 +186,19 @@
       border-color: transparent;
       box-shadow: none;
     }
+
+    &.pressed {
+      background: var(--p-button-color-depressed);
```

Comment:
This seems confusing because we've got pressed above with a pressed var used. Is there anyway to separate these by using a depressed class?

Edit: or is this just an override?

Reply:
This is for primary and critical button variants, and uses this weird css var aliasing to work. This CSS will all get heavily refactored soon, so I'm good with the inelegance.